qcodes/instrument_drivers/tektronix/Keithley_2450.py

```diff
@@ -109,6 +131,22 @@ def clear_trace(self) -> None:
         """
         self.write(":TRACe:CLEar")
 
+    def get_user_number(self) -> int:
+        return self._user_number
```

Comment:
the functionality can be implemented via `set_cmd=None, get_cmd=None` without the need for a private attribute.

Reply:
Nice! I was hoping there was a simpler way to do it. I have implemented as you suggested.

Reply:
I also removed the self._user_number=1 in the __init__ because the user number should be a required input parameter, not with a default value of 1.